packages/input-datepicker/src/LionCalendarOverlayFrame.js

```diff
@@ -94,7 +95,7 @@ export class LionCalendarOverlayFrame extends LocalizeMixin(LitElement) {
   }
 
   __dispatchCloseEvent() {
-    this.dispatchEvent(new Event('close-overlay'), { bubbles: true });
```

Comment:
Well, it wasn't bubbling before, as that syntax is wrong. It would have to be:
```js
this.dispatchEvent(new Event('close-overlay', { bubbles: true }));
```
Since bubbling never happened to begin with because of this syntax error, I'm assuming we can live without it :P

Reply:
Just checked but if this would bubble, it would bubble all the way up to the document.body, as the calendar-overlay-frame would be in the top-most container of its global_overlays "outlet". Probably that's not what you want.